scanner/views.py

```
@@ -0,0 +1,116 @@
+    }
+    return render(request, 'scanner/dashboard.html', context)
+
+
+@csrf_exempt
+@require_http_methods(["POST"])
+def start_scanner(request):
+    try:
+        data = json.loads(request.body) if request.body else {}
+        result = scanner_service.start(data)
+        return JsonResponse(result)
+    except Exception as e:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)